src/util/mod.rs

```diff
@@ -9,14 +11,15 @@ pub use belt::StagingBelt;
 #[repr(align(4))]
 pub struct WordAligned<Bytes: ?Sized>(pub Bytes);
 
-/// Treat the given by slice as a SPIR-V module.
+/// Treat the given byte slice as a SPIR-V module.
 ///
-/// # Errors
+/// # Panic
```

Comment:
This can be changed back to errors if the function is converted to use `Result` instead of `assert!`